src/vm/eventpipebuffermanager.cpp

```diff
@@ -184,7 +201,7 @@ EventPipeBufferList* EventPipeBufferManager::FindThreadToStealFrom()
                 pOldestContainingList = pCandidate;
             }
             // Otherwise, to replace the existing candidate, this candidate must have an older timestamp in its oldest buffer.
-            else if((pOldestContainingList->GetHead()->GetMostRecentTimeStamp().QuadPart) > 
+            else if((pOldestContainingList->GetHead()->GetMostRecentTimeStamp().QuadPart) >
```

Comment:
Nit: Can you please remove the whitespace only changes?  It looks like there are more non-memleak changes.

Reply:
I've removed whitespace changes.